src/distance.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {maxDistance};
+const {
+    assertArray,
+    assertArrayItemsInstanceOf,
+    assertArrayLengthEq,
+    assertArraysOfArrayNotEmpty,
+    assertArrayNotEmpty
+} = require("./typeassert");
+const {transposeArray} = require("./math.js");
+
+/**
+ * @summary Measures the max distance between all criterias.
+ * @param {number[][]} criteria 2d array with students and their criterias
+ * @returns {number} the calculated score
+ */
+function maxDistance(criteria){
+    assertArray(criteria);
+    assertArrayNotEmpty(criteria);
+    assertArrayItemsInstanceOf(criteria, Array);
+    assertArraysOfArrayNotEmpty(criteria);
+    assertArrayLengthEq(...criteria);
+    let transposedCriteria = transposeArray(criteria);
+    let value = 0;
+    for (let i = 0; i < criteria[0].length; i++){
+        let min = Math.min(...transposedCriteria[i]);
+        let max = Math.max(...transposedCriteria[i]);
+        value += max + (-min);
```

Comment:
Formlen for forskellen imellem to tal (distancen) er max - min

Reply:
💯